server/eventHandlers/storage.ts

```diff
@@ -68,24 +67,45 @@ export default class extends IEventHandler {
    * NOTE: this may modify the event.
    */
   public async handle(event: CWEvent) {
-    const chain = event.chain || this._chain
+    let chain = event.chain || this._chain;
 
     event = this.truncateEvent(event);
+
     log.debug(`Received event: ${JSON.stringify(event, null, 2)}`);
-    const shouldSkip = await this._shouldSkip(event);
+
+    // TODO: this entire if statement is unnecessary with new system since if the token is
```

Comment:
why is this necessary to include then?

Reply:
I made that comment since it's code that should be removed if we eventually move on to only using the new CE system. The only purpose of it now is for compatibility with the old CE system.